lib/endpoints/class-wp-json-comments-controller.php

```diff
@@ -6,6 +6,123 @@
 class WP_JSON_Comments_Controller extends WP_JSON_Controller {
 
 	/**
+	 * Register the routes for the objects of the controller
+	 * 
```

Comment:
Extra line in the docblock should be trimmed here, and the description should have a trailing period.